tests/org.jboss.tools.openshift.ui.bot.test/resources/eap64-basic-s2i.json

```diff
@@ -30,13 +30,13 @@
         {
             "description": "Git source URI for application",
             "name": "SOURCE_REPOSITORY_URL",
-            "value": "https://github.com/mlabuda/jboss-eap-quickstarts",
+            "value": "https://github.com/rhopp/jboss-eap-quickstarts",
```

Comment:
One quickstart have modified pom.xml to include openshift profile - I am planning to get rid of this dependency...